src/httpClient/httpClient.ts

```diff
@@ -90,20 +106,37 @@ export default class HttpClient {
     );
 
     this.client.interceptors.response.use(
-      (response) => response,
+      (response) => {
+        if (this.logOptions.enabledLogs.includes(LogType.responses)) {
+          this.logFunction({
+            title: 'HTTP Response',
+            level: 'INFO',
+            method: response.config.method,
+            url: response.config.url,
+            query: response.config.params,
+            request: safeJsonParse(response.config.data, response.config.data),
```

Comment:
`response.config.data` is really just the `request`?

Reply:
I don't understand your question, what do you mean?

If it is about using `response.request` field that doesn't work (it is the HTTP request instance, not the request body which I want here).

Reply:
I just wouldn't expect that `response.config.data` is the request as the name doesn't suggest this

Reply:
This is the naming I've used:
- request body goes into `request`
- response body goes into `response`

Do you have a different idea of how to call these two fields, so that it is more intuitive/according to your expectations?

Reply:
Axios response schema https://github.com/axios/axios#response-schema documents it like this:
```
  // `config` is the config that was provided to `axios` for the request
  config: {},

  // `request` is the request that generated this response
  // It is the last ClientRequest instance in node.js (in redirects)
  // and an XMLHttpRequest instance in the browser
  request: {}
```
So the request is unfortunately not the request body...it might contain it but somewhere deep inside.